src/components/movies/MovieList.vue

```diff
@@ -1,11 +1,12 @@
 <template>
-  <div>
+  <div class="movie-list">
     <MovieCard
-      v-for="movie in $store.getters.movieList"
+      v-for="movie in moviesByCategory"
       :key="movie.id"
       :title="movie.title"
       :img="movie.poster_url"
       :genre="movie.genre.name"
+      :length="movie.length"
```

Comment:
Considering we operate on some `movie` objects anyway - instead of multiple props like `title`, `img`, `genre` and so on we could consider passing the whole movie there:
```suggestion
      v-for="movie in moviesByCategory"
      :key="movie.id"
      :movie="movie"
```

You would need to refactor the `MovieCard` component to only accept some `movie` object prop but this would clean our code up a little bit 😄
What do you think?

Reply:
I wasn't sure if I can pass on the whole object as a prop :) THANK YOU!!